### PR TITLE
Add component to map candlepin pools to capacity

### PR DESCRIPTION
**Pull request by kahowell**
Depends on #59 ... so marking as draft until #59 merged.

### CODE REVIEW

File: src/main/java/org/candlepin/subscriptions/capacity/CandlepinPoolCapacityMapper.java [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+            capacity.setBeginDate(pool.getStartDate());
+            capacity.setEndDate(pool.getEndDate());
+            Integer socketLimit = getSocketLimit(pool);
+            if (products.contains(product)) {
+                capacity.setPhysicalSockets(socketLimit);
+            }
+            if (derivedProducts.contains(product)) {
+                capacity.setVirtualSockets(socketLimit);
+            }
+            return capacity;
```
Comment (mstead) [inline src/main/java/org/candlepin/subscriptions/capacity/CandlepinPoolCapacityMapper.java]: Are we not taking quantity into consideration when determining the capacity?

**Comment by kahowell** [comment]
- 90b814b to f7cde84: rebase against master
- f7cde84: rename method for clarity

**Comment by kahowell** [comment]
@mstead, please take another look when you get a chance.

File: src/main/java/org/candlepin/subscriptions/capacity/CandlepinPoolCapacityMapper.java (hunk 1 above)
Comment (kahowell) [reply src/main/java/org/candlepin/subscriptions/capacity/CandlepinPoolCapacityMapper.java]: We are actually. See latest commit. I chose variable and method names poorly. :-)

**Comment by kahowell** [comment]
b10de6d to 8cb455a: rebase and squash

**Pull request merged**